src/modules/leaderboard.js

```diff
@@ -1,25 +1,34 @@
+import Api from './api.js';
 import Player from './player.js';
 
 export default class Leaderboard {
   constructor() {
     this.leaders = [];
   }
 
-  addScore(user, score) {
+  static async addScore(user, score) {
     const newPlayer = new Player(user, score);
-    this.leaders.push(newPlayer);
+    const response = await Api.post(newPlayer);
+    return response;
   }
 
-  display() {
+  async display() {
```

Comment:
- [ ] I would implore you to use an arrow function so that it would be flow with ES6. It would look like `async display = () => { ... }`. Kindly add it to all your classes

Reply:
Thank you for you suggestion, you done great job recommending how i suppose to do it.
Can you help me if you're chance. Thanks